### PR TITLE
grpc: changelog for first release added

### PR DESCRIPTION
**Pull request by amitpanwar789**
## Related Issues
[zaproxy/zaproxy#7695](https://github.com/zaproxy/zaproxy/issues/7695)

## Checklist
- [ ] Update help
- [ ] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [ ] Write tests
- [ ] Check code coverage
- [x] Sign-off commits
- [ ] Squash commits
- [ ] Use a descriptive title

For more details, please refer to the [developer rules and guidelines](https://www.zaproxy.org/docs/developer/dev-rules-and-guidelines/).

### CODE REVIEW

File: addOns/grpc/CHANGELOG.md [hunk 1]
```diff
@@ -5,5 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
-- First version.
-
+## 0.1.0 - 2024-05-17
+- First Version
```
Comment (thc202) [inline addOns/grpc/CHANGELOG.md:9]: `### Added`

File: addOns/grpc/CHANGELOG.md [hunk 2]
```diff
@@ -5,5 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
+- Features
+  - ProtoBuf Message Decoding and Encoding
```
Comment (thc202) [inline addOns/grpc/CHANGELOG.md:11]: `... in the message view panels.`

**Review by psiinon — APPROVED** [review]

File: addOns/grpc/CHANGELOG.md [hunk 3]
```diff
@@ -5,5 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
-- First version.
-
+## 0.1.0 - 2024-05-17
```
Comment (kingthorin) [inline addOns/grpc/CHANGELOG.md:8]: You should leave this, it'll be done by the release workflow.

Comment (kingthorin) [reply addOns/grpc/CHANGELOG.md:8]: I mean leave the unreleased heading, remove this one, leave the added and notes

File: addOns/grpc/CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## Unreleased
 
 ## 0.1.0 - 2024-05-17
```
Comment (thc202) [inline addOns/grpc/CHANGELOG.md:8]: This line should be removed, that will be done automatically during the release process.

File: addOns/grpc/CHANGELOG.md (hunk 3 above)
Comment (thc202) [reply addOns/grpc/CHANGELOG.md:8]: The "First version" entry can be removed too, that's implicit by being the first release.

Comment (thc202) [reply addOns/grpc/CHANGELOG.md]: The `## Unreleased` should be kept.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**